### ISSUE TITLE
Max value for zoom / camera offset

### ISSUE THREAD

**Issue by eriq123** [Smithsonian/dpo-voyager#151]
As users zoom out, we need the max values of the camera offset specifically the z-coordinate to disable our zoom-out button accurately when the max zoom is reached.

The current workaround is to save an initial offset then get the new offset and compare the z-coordinate. The z-coordinate does not change if the maximum value is reached so if the initial and the new offset are the same I save it as the max zoom value.

```
let maxZoomValue = '';
const zoomIncrement = 25;

var voyagerElement = document.getElementById("voyager");
const initialOffset = voyagerElement.getCameraOffset();
voyagerElement.setCameraOffset(x.value, y.value, z.value + zoomIncrement);

const newOffset = voyagerElement.getCameraOffset();

if (initialOffset[2].value === newOffset[2].value) {
  maxZoomValue = newOffset[2].value;
}
```

it would be great if we can access an API like `voyagerElement.getCameraOffset('max')` or something similar.

**Comment by gjcope** [comment]
Thanks @eriq123 , good idea. We've added an optional 'type' parameter to getCameraOffset and getCameraOrbit. It's checked in here: https://github.com/Smithsonian/dpo-voyager/tree/dev-bugfix and will be included in the next release (end of the month). Keep in mind the max offset is calculated at runtime based on the scene bounds and canvas aspect ratio so you will want to wait a couple of cycles to grab the final value.

**Comment by gjcope** [comment]
Released in v0.20.0
